apps/3000-home/next.config.js

```diff
@@ -23,18 +48,18 @@ const nextConfig = {
     // used for testing build output snapshots
 
     const remotes = {
-      shop: createDelegatedModule(require.resolve('./remote-delegate.js'), {
-        remote: `shop@http://localhost:3001/_next/static/${
-          isServer ? 'ssr' : 'chunks'
-        }/remoteEntry.js`,
-      }),
+      // shop: createDelegatedModule(require.resolve('./remote-delegate.js'), {
+      //   remote: `shop@http://localhost:3001/_next/static/${
+      //     isServer ? 'ssr' : 'chunks'
+      //   }/remoteEntry.js`,
+      // }),
       // checkout: createDelegatedModule(require.resolve('./remote-delegate.js'), {
       //   remote: `checkout@http://localhost:3002/_next/static/${isServer ? 'ssr' : 'chunks'}/remoteEntry.js`,
       // }),
 
-      // shop: `shop@http://localhost:3001/_next/static/${
```

Comment:
there is a problem with Federation Plugin, the runtime modules are getting registered more than once causing share scope to attempt to reinitialize.